eogrow/pipelines/download.py

```diff
@@ -67,7 +68,7 @@ class RescaleSchema(BaseSchema):
 
 class PostprocessingRescale(BaseSchema):
     rescale_schemas: List[RescaleSchema] = Field(
-        default_factory=list, description="Specify different ways to rescale features"
+        default_factory=List[RescaleSchema], description="Specify different ways to rescale features"
```

Comment:
missed update to `list_factory`?

Reply:
yes, apparently tests dont touch that :see_no_evil: